www/wp-content/object-cache.php

```diff
@@ -189,10 +189,6 @@ function delete($id, $group = 'default') {
 	}
 
 	function flush() {
-		// Don't flush if multi-blog.
-		if ( function_exists('is_site_admin') || defined('CUSTOM_USER_TABLE') && defined('CUSTOM_USER_META_TABLE') )
```

Comment:
That will still allow cache flushing logic to work in Quickstart, which I'd like to avoid. What if we set a constant in the `wp-test-config.php` and looked for that?